presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -144,6 +160,7 @@ def analyze(self, correlation_id, text, entities, language, all_fields):
                 results.extend(current_results)
 
         results = AnalyzerEngine.__remove_duplicates(results)
+        results = AnalyzerEngine.__remove_low_scores(results, score_threshold)
```

Comment:
consider: adding something like:
total results: X, dropped due to low score: Y

maybe it will be useful when debugging in the future

Reply:
I will put this line before the app tracer line. This will log all the results prior to removal. Would that be sufficient?